.travis.yml

```diff
@@ -1,6 +1,5 @@
-language: c
-dist: xenial
```

Comment:
Do we need to actually change these?  I think the individual blocks can override the global settings.  Is there an advantage to one vs. the other?

Reply:
I just checked this and you are right; I've moved dist xenial back to global scope. My branch (and this PR) have been updated.  Thanks